src/Moryx.CommandCenter.Web/src/common/components/Menu/RoutingMenuItem.tsx

```diff
@@ -44,10 +44,20 @@ function RoutingMenuItem(props: MenuItemProps) {
         }
     };
 
-    const isActive = isOpened(location);
+    const isActive = (location: Location): boolean => {
```

Comment:
What is the difference between isOpened and isActive semantically here? Are these two different concepts or should your addition be a fix for the isOpened stuff that was already there? 🤔

Reply:
It's two different concepts: `isOpened` tells if a complete path is opened, while `isActive` indicates, if the 'base' part is 'active'. If you navigate to `modules/ProcessEngine/configuration`, these are the results when `modules/ProcessEngine` is assumed the base:

```
modules/ProcessEngine/configuration - isActive = true, isOpened = true
modules/ProcessEngine/console - isActive = true, isOpened = false
modules/ProcessEngine - isActive = true, isOpened = false
modules/Orders - isActive = false, isOpened = false
 ```

Yes, I skipped writing tests, since there is no infrastructure right now